Core/PoTItem.cs

```diff
@@ -77,9 +77,10 @@ public virtual int ItemLevel
 		get => InternalItemLevel;
 		set => InternalItemLevel = value;
 	}
+	
+	public virtual string Description => string.Empty;
+	public virtual string AltUseDescription => string.Empty;
 
-	public virtual string Description => "";
-	public virtual string AltUseDescription => "";
```

Comment:
Curious; What told you to do this? Resharper? Some other tool? Or just personal knowledge?

Reply:
I *believe* that using string.Empty uses a premade empty string, which is marginally more efficient. Not like it's a major performance sink or anything, I did this only because I did some other changes, of which I reverted aside from the string.Empty change.